txp/Pos/Txp/Toil/Monad.hs

```diff
@@ -223,7 +223,7 @@ getStake id =
     (<|>) <$> lift (use (gtsStakesView . svStakes . at id)) <*> baseLookup id
   where
     baseLookup :: StakeholderId -> GlobalToilM (Maybe Coin)
-    baseLookup i = lift $ lift $ lift $ Free $ StakesLookupF i pure
+    baseLookup i = lift $ lift $ lift $ F $ \kPure kFree -> kFree (StakesLookupF i kPure)
```

Comment:
To be honest, I forgot about `powerLift`. Now I checked it and it doesn't work out of box:
```
/home/gromak/Work/cardano-sl/txp/Pos/Txp/Toil/Monad.hs:227:20: error:
    • Overlapping instances for Pos.Util.Util.PowerLift
                                  (F StakesLookupF) (F StakesLookupF)
        arising from a use of ‘powerLift’
      Matching instances:
        instance [overlapping] forall k (m :: k -> *).
                               Pos.Util.Util.PowerLift m m
          -- Defined in ‘Pos.Util.Util’
        ...plus one instance involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In the expression:
        powerLift $ F $ \ kPure kFree -> kFree (StakesLookupF i kPure)
      In an equation for ‘baseLookup’:
          baseLookup i
            = powerLift $ F $ \ kPure kFree -> kFree (StakesLookupF i kPure)
      In an equation for ‘getStake’:
          getStake id
            = (<|>) <$> lift (use (gtsStakesView . svStakes . at id))
              <*> baseLookup id
            where
                baseLookup :: StakeholderId -> GlobalToilM (Maybe Coin)
                baseLookup i
                  = powerLift $ F $ \ kPure kFree -> kFree (StakesLookupF i kPure)
```

To be honest, I don't want to spend much time to figure out how to make `powerLift` work here (I can use `-fprint-potential-instances` and then think about it, but I don't want to, because it's minor).